packages/playground/wordpress/src/boot.ts

```diff
@@ -190,13 +197,13 @@ export async function bootWordPress(options: BootOptions) {
 	php.defineConstant('WP_HOME', options.siteUrl);
 	php.defineConstant('WP_SITEURL', options.siteUrl);
 
-	// @TODO Assert WordPress core files are in place
-
 	// Run "before database" hooks to mount/copy more files in
 	if (options.hooks?.beforeDatabaseSetup) {
 		await options.hooks.beforeDatabaseSetup(php);
 	}
 
+	// @TODO Assert WordPress core files are in place
```

Comment:
Moved this because worker-thread uses the `beforeDatabaseSetup` hook to setup files from OPFS:
https://github.com/WordPress/wordpress-playground/blob/d6a90afd9567bd7a5eaa6010193dc471788911cc/packages/playground/remote/src/lib/worker-thread.ts#L235

Perhaps it should be a different hook for OPFS setup, but that is what is used now.